README.md

```diff
@@ -33,6 +33,32 @@ setupSinonSandbox();
 
 This will automatically wire-up the sandbox `sinon.sandbox.create` and `sandbox.restore` methods to QUnit `testStart` and `testDone` respectively.
 
+### Accessing Sinon from Within Tests
+
+The `ember-sinon-sandbox` addon converts the global sinon object to a sandbox by default for each test. Additionally, in your tests you will be able to access the same sandboxed version of sinon via the `this.sandbox` property available within the test's scope:
+
+```js
+test('very important test happening here', function(assert) {
+  const stub = this.sandbox.stub();
```

Comment:
I think that we shouldn't use `stub` without arguments in the example

Reply:
Hehe. You'd think I'd have learned from our chat this morning. Will change.